### ISSUE TITLE
Issue with "EventTarget.addEventListener()": (short summary here please)

### ISSUE THREAD

**Issue by bytesized** [mdn/content#3249]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

#### What information was incorrect, unhelpful, or incomplete?

The "Syntax" section contains the text "This API has not been standardized.", which is fine, but it seems to be in the middle of the syntax specification, like this:

```
target.addEventListener(type, listener [, useCapture, wantsUntrusted 
    This API has not been standardized.
    
    
]); // Gecko/Mozilla only
```
I'm not sure how this ought to look, but it doesn't seem right to me that the sentence comes between the open and close parentheses.

#### Specific section or headline?

["Syntax"](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#syntax)

#### What did you expect to see?

I expected there to not be a stray sentence in the middle of the syntax specification.

#### Did you test this? If so, how?
N/A

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/eventtarget/addeventlistener`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/eventtarget/addeventlistener/index.html
* Last commit: https://github.com/mdn/content/commit/f37c24de21200c64921064e536ea591d71df6cb3
* Document last modified: 2021-03-02T22:00:49.000Z

</details>

**Comment by CreaTorAlexander** [comment]
Also it uses the old Syntax *[, useCapture, wantsUntrusted]*

**Comment by hamishwillee** [comment]
Thanks, fixed up in #3285 (error was an inline macro inside a pre tag)
